hack/update-rhcos-bootimage.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+import os,sys,json,argparse
+import urllib.request
+
+dn = os.path.abspath(os.path.dirname(sys.argv[0]))
+
+parser = argparse.ArgumentParser()
+parser.add_argument("meta", action='store')
+args = parser.parse_args()
+
+with urllib.request.urlopen(args.meta) as f:
+    meta = json.load(f)
+newmeta = {}
+for k in ['amis', 'images', 'buildid', 'oscontainer',
+          'ostree-commit', 'ostree-version']:
+    newmeta[k] = meta[k]
+with open(os.path.join(dn, "../data/data/rhcos.json"), 'w') as f:
+    json.dump(newmeta, f, indent=4)
```

Comment:
Can you add [`sort_keys=True`][1] for stability?

[1]: https://docs.python.org/3/library/json.html#json.dump

Reply:
Done locally, but going to wait for this `e2e-aws` run to finish before pushing to see whether that last run was a fluke or not.

Reply:
`e2e-aws` passed, now updated with the `sort_keys=True` :arrow_down: